### PR TITLE
homepage added

### PR DESCRIPTION
**Pull request by palash23jain**
Screenshot 2022-02-21 at 2 05 45 AM

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.cric-quiz-intro{
+    font-size: var(--fontSize3);
+    padding: var(--spacing3);
+}
+
+.cric-quiz-intro{
+    width: 35rem;
+    margin: 0 auto;
+}
+
```
Comment (BelideNikhil) [inline style.css]: You can combine these

File: style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.img-wrapper{
+    margin: var(--spacing6) 0;
+    display: flex;
+    justify-content: center;
+}
+
+img {
+    width: 35rem;
+    height: 30rem;
+}
```
Comment (BelideNikhil) [inline style.css]: this will effect all images globally, so add a class

File: common-css/main.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    width: 30rem;
+}
+
+.navbar-container{
+    justify-content: space-around;
+}
+
+.navbar-item{
+    cursor: pointer;
+}
```
Comment (pritam-kr) [inline common-css/main.css:23]: remove these red mark

File: style.css (hunk 2 above)
Comment (palash23jain) [reply style.css]: done the changes as suggested by you. Please check now

**Review by BelideNikhil — COMMENTED** [review]
Try Compressing your images or use webp as a good practice
Right now, the image size is too large
Rest look good!

**Review by pritam-kr — APPROVED** [review]
Good to merge

**Pull request merged**